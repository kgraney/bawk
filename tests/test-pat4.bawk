a = 100;

/89 foo:int1 4e47/ { /* should match */
	print(foo);
}

/89504e47/ { /* should match */
	print(20);
}


print(30);
