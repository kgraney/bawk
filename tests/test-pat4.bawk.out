80
20
30
